### PR TITLE
Code Style: Fix Constant naming

### PR DESCRIPTION
**Pull request by david-allison-1**
Should be ALL_CAPS_WITH_UNDERSCORES: https://github.com/ankidroid/Anki-Android/wiki/Code-style#constant-final-variables-names-must-be-all-uppercase-using-underscore-to-separate-words

> Constant (final variables) names must be all uppercase using underscore to separate words.

I did this with a lint check (on `public static final`), this is checked in to a personal branch: `lint_constant_done` and will be added after https://github.com/ankidroid/Anki-Android/issues/8387 is done to avoid problems

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/importer/Anki2Importer.java [hunk 1]
```diff
@@ -30,7 +30,6 @@
 import com.ichi2.libanki.Decks;
 import com.ichi2.libanki.Media;
 import com.ichi2.libanki.Model;
-import com.ichi2.libanki.Note;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/importer/Anki2Importer.java:33]: revert

File: AnkiDroid/src/main/java/com/ichi2/libanki/StdModels.java [hunk 2]
```diff
@@ -132,13 +132,13 @@ public String getDefaultName() {
...
+    public static final StdModels[] STD_MODELS =
     {
-        basicModel,
-        basicTypingModel,
-        forwardReverseModel,
-        forwardOptionalReverseModel,
-        clozeModel,
+            BASIC_MODEL,
+            BASIC_TYPING_MODEL,
+            FORWARD_REVERSE_MODEL,
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/StdModels.java:139]: hmm, spacing changed

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java [hunk 3]
```diff
@@ -94,8 +93,8 @@
 import static com.ichi2.async.TaskManager.setLatestInstance;
 import static com.ichi2.libanki.Card.deepCopyCardArray;
 import static com.ichi2.libanki.Undoable.*;
-import static com.ichi2.utils.BooleanGetter.False;
-import static com.ichi2.utils.BooleanGetter.True;
+import static com.ichi2.utils.BooleanGetter.FALSE;
+import static com.ichi2.utils.BooleanGetter.TRUE;
```
Comment (Arthur-Milchior) [inline AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java:97]: That's really feel strange to see `FALSE` in all upper case. I know very well why it must be a constant and can't just be the Boolean or a boolean, but still it do not feel right to me to read it this way even if it's the right way. (And I just checked, `Boolean.FALSE` is the actual False constant)

**Review by Arthur-Milchior — APPROVED** [review]
Quite sorry, it seems more than half of the changes were my faults

**Pull request merged**